peregrine/iqgen/bits/doppler_linear.py

```diff
@@ -0,0 +1,231 @@
+# Copyright (C) 2016 Swift Navigation Inc.
+#
+# This source is subject to the license found in the file 'LICENSE' which must
+# be be distributed together with this source. All other rights reserved.
+#
+# THIS CODE AND INFORMATION IS PROVIDED "AS IS" WITHOUT WARRANTY OF ANY KIND,
+# EITHER EXPRESSED OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE IMPLIED
+# WARRANTIES OF MERCHANTABILITY AND/OR FITNESS FOR A PARTICULAR PURPOSE.
+from peregrine.iqgen import if_iface
+from scipy.stats.vonmises_cython import numpy
+
+"""
+The :mod:`peregrine.iqgen.doppler_linear` module contains classes and functions
+related to generation of signals with linearily changing doppler.
+
+"""
+
+import math
+import scipy.constants
+
+class Doppler(object):
```

Comment:
I think all my comments from `doppler_const.py` apply here as well. Could we make `doppler_const` a subclass of the `Doppler` class and have them in the same file? There's a lot of overlapping code and it seems like they should be part of the same file/module.